frontend/src/bundles/auth/components/AdditionalForm.vue

```diff
@@ -203,6 +240,21 @@ export default {
     };
   },
 
```

Comment:
Consider a possibility splitting the form into few smaller parts

Reply:
@AlexeyTryapko, what do you think?

Reply:
Agree